src/Sarif.Viewer.VisualStudio/Themes/CallTrees.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:local="clr-namespace:Microsoft.Sarif.Viewer.Themes"
+                    xmlns:shell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.14.0">
+
+    <ResourceDictionary.MergedDictionaries>
+        <ResourceDictionary Source="pack://application:,,,/Microsoft.Sarif.Viewer;component/Themes/DefaultStyles.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+
+    <DataTemplate x:Key="CallTreeTemplate">
+        <ListView ItemsSource="{Binding}">
+            <ListView.ItemTemplate>
+                <DataTemplate>
+                    <TreeView ItemsSource="{Binding TopLevelNodes}">
+                        <TreeView.ItemTemplate>
+                            <HierarchicalDataTemplate DataType="x:Type local:CallTreeNode" ItemsSource="{Binding Children}">
+                                <TextBlock Text="{Binding Location.Kind}" />
```

Comment:
@jericahuang @kschecht Although it won't be obvious to you without some deeper understanding of WPF, this is the place where we'll put all the fancy data about each location. Here's where we'll decide what text to display, and what color the font and the background will be, to match SDV's display. And we'll have to figure out what how to display the "state data" for each location.